cmd/gcp-controller-manager/gcp_config.go

```diff
@@ -19,13 +19,16 @@ limitations under the License.
 package main
 
 import (
+	"bufio"
 	"context"
 	"crypto/x509"
 	"fmt"
-	"k8s.io/cloud-provider-gcp/providers/gce"
+	"os"
 	"sort"
 	"strings"
 
+	"k8s.io/cloud-provider-gcp/providers/gce"
+
```

Comment:
remove extra space

Reply:
Ok, will do, this change is auto generated by style check